docs/odk-workflows/ManageDocumentation.md

```diff
@@ -38,9 +38,10 @@ The documentation is _not_ automatically updated from the Markdown, and needs to
    If everything was successful, you will see a message similar to this one:
 
    ```
-   INFO    -  Your documentation should shortly be available at: https://information-artifact-ontology.github.io/ontology-metadata/ 
+   INFO    -  Your documentation should shortly be available at: https://information-artifact-ontology.github.io/noname/ 
```

Comment:
What did happen with the ontology name?

Reply:
Are you going to discard the changes on this and ReleaseWorkflow files?

Reply:
ooooookeeeeeeee...

Reply:
Thank you! 😊